pennylane/templates/broadcast.py

```diff
@@ -23,9 +23,11 @@
 from collections import Iterable
 
 from pennylane.templates.decorator import template
-from pennylane.templates.utils import check_wires, check_type, get_shape, check_is_in_options
+from pennylane.templates.utils import check_type, get_shape, check_is_in_options
+from pennylane.wires import Wires
 
 
```

Comment:
Most changes in this module address the fact that patterns (lists of wires) are now list of Wire objects.